index.html

```diff
@@ -0,0 +1,169 @@
+            <div class="program-list">
+                <!-- Card 1 -->
+                <div class="program-card">
+                    <img src="imgs/distribution-icon.png" alt="Marketplace">
+                    <h3 class="program-title">Distribution</h3>
+                    <p>“The best supply chain is one that has no beginning and no end.”</p>
+                </div>
+                <!-- Card 2 -->
+                <div class="program-card">
+                    <img src="imgs/digital-marketing.png" alt="Affiliate">
+                    <h3 class="program-title">Marketing</h3>
+                    <p>“Business has only two functions — marketing and innovation.”</p>
+                </div>
+                <!-- Card 3 -->
+                <div class="program-card">
+                    <img src="imgs/promotion.png" alt="blog">
+                    <h3 class="program-title">Promotion</h3>
+                    <p>“It's not what you sell that matters as much as how you sell it!”</p>
+                </div>
+                <!-- Card 4 -->
+                <div class="program-card">
+                    <img src="imgs/educate.png" alt="Education">
+                    <h3 class="program-title">Education</h3>
+                    <p>“Education is one thing no one can take away from you.”</p>
+                </div>
+                <!-- Card 5 -->
+                <div class="program-card">
+                    <img src="imgs/payment.png" alt="Payment">
+                    <h3 class="program-title">Payment</h3>
+                    <p>Our tools maskes it easier to scale your business online.</p>
+                </div>
+            </div>
+            <a href="#" class="program-text" type="button"> SEE THE WHOLE PROGRAM</a>
+        </section>
+        <section class="speakers-container" id="news">
+            <h2>Featured speakers</h2>
+            <div class="speakers"></div>
+            
+        </section>
+
+        <section class="sponsor" id="sponsor">
+            <h2>Partners</h2>
+            <ul class="sponsor-brand">
+                <li class="sponsor-list">
+                    <img src="imgs/microverselogo.webp" alt="sponsor1" class="spon1">
+                </li>
+
+                <li class="sponsor-list">
+                    <img src="imgs/cisco-logo-transparent.png" alt="sponsor2" class="spon1">
+                </li>
+
+                <li class="sponsor-list">
+                    <img src="imgs/FreeCodeCamp_logo.png" alt="sponsor2" class="spon1">
+                </li>
+            </ul>
+        </section>
+    </main>
+
+    <!-- Footer  -->
+    <footer class="info" id="campaign">
+        <div class="info-content">
+            <a href="#home" class="nav-brands">Digital Adept</a>
+        <span>TM</span>
+        </div>
+        <p class="info-credit">2023 Proud2Pay Marketing Campaign And Events Made with ❤️  By <a href="#">Henry Inspired</a>  Inspire By <a href="#">Cindy Shin</a>💎</p>
```

Comment:
- Kindly as it was required to buy the previous reviewer it should be better to add a footer with appropriate design and disposition and make sure you center the content   `<h2 class="wit">shyn designs</h2>` I suggest you use the logo that you did use in the navbar. it will make your design stick more to the design provided by microverse :+1: 

- Kindly make sure the footer is displayed as expected as I show you in the picture below your footer is not aligned correctly  fix that  it will make your design stick more to the design provided by microverse :+1: 

## design provides 

![image](https://user-images.githubusercontent.com/95297251/204106167-3ec1d3ca-049e-4ef0-bbcd-94c7889d3263.png)


## Your design 

![image](https://user-images.githubusercontent.com/95297251/212410830-0906655d-3a52-4fc4-9f20-05143f9d194d.png)